enterprise_subsidy/apps/api_client/enterprise.py

```diff
@@ -178,3 +185,22 @@ def cancel_fulfillment(self, enterprise_fulfillment_uuid):
                 f'{enterprise_fulfillment_uuid}. Failed with error: {exc}'
             )
             raise exc
+
+    def fetch_recent_unenrollments(self):
+        """
+        Fetches enterprise enrollment objects that have been unenrolled within the last 24 hours.
+        """
+        unenrolled_subsidies_url = self.enterprise_fulfillment_unenrollments_url()
+        unenrolled_cutoff = (datetime.now(tz=timezone.utc) - timedelta(hours=24 * 7)).isoformat()
+        try:
+            response = self.client.get(
+                unenrolled_subsidies_url,
+                params={"unenrolled_after": unenrolled_cutoff}
+            )
+            response.raise_for_status()
+            return response.json()
+        except requests.exceptions.HTTPError as exc:
```

Comment:
Nit: consier using the even more general `requests.exceptions.RequestException`, which will also catch things like connection errors.